examples/chip-tool/commands/common/CHIPCommand.cpp

```diff
@@ -83,6 +86,41 @@ CHIP_ERROR GetAttestationTrustStore(const char * paaTrustStorePath, const chip::
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR GetRevocationSet(const char * revocationSetPath, const chip::Credentials::RevocationSet ** revocationSet)
+{
+    bool failed = false;
+
+    if (revocationSetPath == nullptr)
+    {
+        revocationSetPath = getenv(kRevocationSetPathVariable);
+    }
+
+    static chip::Credentials::JSONFileRevocationSet fileRevocationSet{ revocationSetPath };
+
+    if (revocationSetPath == nullptr)
```

Comment:
Shouldn't the null-check be before we use the value?

Reply:
No. If no revocationSetPath is given via cmdline, we shall fetch the revocationSetPath from an environment variable. This is the same logic as used in GetAttestationTrustStore method above

Reply:
What does the env var bit have to do with my comment?  The value (which may be null) is used on line 98.  The null check is on line 100.

Reply:
Addressed in the latest commit.